waywiser_twist_safety/src/rclcpp_components/emergency_stop_monitor_core.cpp

```diff
@@ -92,6 +85,103 @@ void EmergencyStopMonitor::twist_watchdog_callback()
 
   twist_watchdog_timer_->cancel();
 }
+
+std::string EmergencyStopMonitor::uuid_to_string(const rclcpp_action::GoalUUID & uuid)
+{
+  std::ostringstream oss;
+  oss << std::hex << std::setfill('0');
+  for (const auto & byte : uuid) {
+    oss << std::setw(2) << static_cast<int>(byte);
+  }
+  return oss.str();
+}
+
+rclcpp_action::GoalResponse EmergencyStopMonitor::set_state_goal_request_callback(
+  const rclcpp_action::GoalUUID & uuid,
+  std::shared_ptr<const waywiser_twist_safety::action::SetEmergencyStopState::Goal> goal)
+{
+  // (void)uuid;
+  if (goal->target_state == EmergencyStopState::EMERGENCY_STOP_STATE_ACTIVE) {
```

Comment:
Using switch case instead of if .. else if ... else whenever enums like EmergencyStopState are handled would improve readability in my opinion.